src/test/java/cohen/worldcitysearch/WorldCitiesListTest.java

```diff
@@ -23,12 +23,10 @@ public void getClosestCity() throws IOException
     {
         WorldCitiesList cities = new WorldCitiesList();
 
-        WorldCity actual = new WorldCity("Chicago", 41.8375, -87.6866);
-
         WorldCity result = cities.getClosestCity(41.83, -87.6);
 
-        assertEquals(result.city(), actual.city());
-        assertEquals(result.lat(), actual.lat());
-        assertEquals(result.lon(), actual.lon());
+        assertEquals(result.city(), "Chicago");
```

Comment:
The first parameter is what is expected, the second parameter is actual.
So switch these.